audio/chmap.h

```diff
@@ -124,7 +129,7 @@ void mp_chmap_get_reorder(int src[MP_NUM_CHANNELS], const struct mp_chmap *from,
 int mp_chmap_diffn(const struct mp_chmap *a, const struct mp_chmap *b);
 
 char *mp_chmap_to_str_buf(char *buf, size_t buf_size, const struct mp_chmap *src);
-#define mp_chmap_to_str(m) mp_chmap_to_str_buf((char[64]){0}, 64, (m))
+#define mp_chmap_to_str(m) mp_chmap_to_str_buf((char[128]){0}, 128, (m))
```

Comment:
Can this not directly be defined in terms of `MP_NUM_CHANNELS`?

Reply:
This particular one could be, because the worse case is `xxx-` times number of channels, minus the last dash. But frankly there are two other places, were it is not obvious what the size will be, i.e. wasapi adds some more info after, so for consistency I left it as is.